src/main/java/br/com/openenade/api/modalidade/Modalidade.java

```diff
@@ -0,0 +1,5 @@
+package br.com.openenade.api.modalidade;
+
+public enum Modalidade {
+    EDUCACAO_PRESENCIAL, EDUCACAO_A_DISTANCIA
```

Comment:
@Icaro-Lima please use values in the Enum to Spring be able to map a String to Enum, you can use this [reference](https://github.com/paulofelipefeitosa/pre-matricula-backend/blob/master/src/main/java/br/edu/ufcg/computacao/psoft/prematriculabackend/models/course/PPC.java).